reV/handlers/outputs.py

```diff
@@ -111,15 +119,14 @@ def writable(self):
 
         Returns
         -------
-        bool
+        writable : bool
             Flag if mode is writable
         """
+        writable = True
         mode = ['a', 'w', 'w-', 'x']
         if self._mode not in mode:
-            raise HandlerRuntimeError('mode must be writable: {}'
-                                      .format(mode))
-
-        return True
+            writable = False
+        return writable
```

Comment:
super nit-picky but can you add a line before the return for read-ability

Reply:
Done and also changed the temp var to is_writable so it doesn't look conflicted with the property/method name.